package.json

```diff
@@ -17,6 +17,7 @@
         "stop-test-server": "stop-autorest-testserver",
         "debug": "node --inspect-brk ./dist/src/main.js",
         "generate-swaggers": "ts-node ./test/utils/test-swagger-gen.ts",
+        "clone:specs": "git clone https://github.com/Azure/azure-rest-api-specs.git ./.tmp/specs",

```

Comment:
Does it make sense to do a shallow clone instead of a full clone to limit disk space usage/time to clone? You can use `--depth` to limit how many changes to pull down, and `--no-single-branch` to apply that depth to each branch. I don't know what happens though if you attempt to checkout a commit that's outside this depth though, so depends on what the behavior is with that, and if we only need to reference branches or arbitrary commits.

Reply:
Good point, let me play a bit with this and see how it works out

Reply:
I can do `depth=1` and `--single-branch` . And since we know the specific branch/commit we want we can fetch each of them. That would cut our disk usage from ~60MB to ~11MB